src/main/java/org/kpmp/repositoryDataset/RepositoryFileDatasetRepository.java

```diff
@@ -12,4 +13,9 @@ public interface RepositoryFileDatasetRepository
 
     List<RepositoryFileDataset> findAll();
 
+    @Query (value = "SELECT MAX(release_version) AS max FROM ar_file_info", nativeQuery = true)
+    public String max();
+
+    @Query (value = "SELECT release_version FROM ar_file_info", nativeQuery = true)
```

Comment:
Is this method being used? It doesn't look like it. Let's remove it if it is not needed.